### ISSUE TITLE
How to use matchPath with route params?

### ISSUE THREAD

**Issue by damianesteban** [marblejs/marble#22]
Based on the tests and documentation it seems like this should work, but I'm getting a 404.

What am I missing?

```
const findTag$: Effect = request$ =>
    request$
      .pipe(
        matchPath('/:id'),
        matchType('GET'),
        map(req => req.params.id),
        switchMap(FakeService.findTag),
        map(response => ({ body: response }))
      );
```

Great library by the way - nice and lightweight, composable and readable.

**Comment by JozefFlakus** [comment]
Hi thanks for feedback!

Documentation says that `matchPath('/:id')` is able to match parametrized routes, but right now version `0.2.x` doesn't intercept those parameters from request url. Intercepted url parameters and queries will be available in version `0.3.x` which will be available soon. Stay tuned!

Sorry, I should clarify it more in `README`.

**Comment by damianesteban** [comment]
Thank you for the clarification.

**Comment by JozefFlakus** [comment]
Hi @damianesteban!
We've just released version `0.3.0` (see changelog: https://github.com/marblejs/marble/releases/tag/v0.3.0)

There is one breaking change, but since we are still before `1.0.x` we had to do this as fast as possible.
